scripts/WinInstallFiles/LaunchJMRI.nsi

```diff
@@ -25,6 +25,10 @@
 ; -------------------------------------------------------------------------
 ; - Version History
 ; -------------------------------------------------------------------------
+; - Version 0.1.31.0
```

Comment:
On line 185, you have version 0.1.30.0
Shouldn't these version numbers be the same?

Reply:
Ahhhhh. Yes .